src/integrationTest/resources/environment.properties

```diff
@@ -2,6 +2,8 @@ database.driverClassName=org.testcontainers.jdbc.ContainerDatabaseDriver
 database.url=jdbc:tc:postgresql:9.6-alpine://localhost/claimstore
 
 pdfService.baseUrl = http://some-test-host/api/v1/pdf-generator
+document_management.api_gateway.url = http://document-management-gateway-api
```

Comment:
Would be nice to get property names consistent... This one introduces a 3rd pattern 😉